### ISSUE TITLE
Inconsistent line width

### ISSUE THREAD

**Issue by LAZER555** [SoftFever/OrcaSlicer#6858]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### OrcaSlicer Version

2.2.0 - beta

### Operating System (OS)

Windows

### OS Version

11

### Printer

elegoo neptune 2s with klipper and a lot of upgrades

### How to reproduce

I don't know why, but Orca gives me incosistent line width. It's the worst on a first layer. While Cura always has identical consistent lines. I have used Orca for a long time, but now i have to return to Cura, even though orca is so much more convenient and better, but i'm tired of this inaccuracies. I've never had good first layers in Orca. Maybe it's related to an issue that somebody here brought up that it would be great to have flow control over walls and infills.
I have the same line width settings in both slicers. In orca i have classic engine.

### Actual results

In Orca
![2024-09-21 16-14-52](https://github.com/user-attachments/assets/86f8ac79-64c0-4e79-bd8d-19693e6b95f4)

### Expected results

In Cura
![2024-09-21 16-15-16](https://github.com/user-attachments/assets/f37d6b44-00c2-4ccc-8ff2-631c1d3cde26)

### Project file & Debug log uploads

[medal1.zip](https://github.com/user-attachments/files/17084210/medal1.zip)
[log.zip](https://github.com/user-attachments/files/17084211/log.zip)

### Checklist of files to include

- [X] Log file
- [X] Project file

**Comment by vgdh** [comment]
looks like the nozzle is too close to the bed. May be you have some z-offset configured in cura?

**Comment by LAZER555** [comment]
> looks like the nozzle is too close to the bed. May be you have some z-offset configured in cura?

No. Just the same catrographer scan. The problem is not only the first layer. The lines are always inconsistent. It's like flow changes depending on the speed, but cura can compensate for it. I don't know

**Comment by cubitik** [comment]
Are you using Arachne as wall generator? Have you tried using Classic as a wall generator?

**Comment by LAZER555** [comment]
> Are you using Arachne as wall generator? Have you tried using Classic as a wall generator?

Yes, i use classic generator
